File: CVFCS/pack1/DanhSachDinhKe.java
package pack1;
import java.util.*;
import java.text.*;

public class DanhSachDinhKe{
  ArrayList<Integer> adj;
  public DanhSachDinhKe(){
    adj = new ArrayList<Integer>();
  } 
  public void push(int x){
    adj.add(x);
  }
  public int get(int id){
    return adj.get(id);
  }
  public void show(){
    System.out.println(adj);
  }
  public int size(){
    return adj.size();
  }
}

File: CVFCS/pack1/DanhSachDinhKe.py
class DanhSachDinhKe:
	def __init__(self) -> None:
		self.adj = []
	
	def push(self, x):
		self.adj.append(x)
	
	def get(self, id):
		return self.adj[id]

	def show(self):
		print(self.adj)
	
	def size(self):
		return len(self.adj)
